README.md

```diff
@@ -98,9 +98,41 @@ but it may be less compatible with the existing Jupyter ecosystem.
 
 Tauri, React, Rust.
 
-Making an alternate frontend is only possible due to the moumental engineering
+Making an alternate frontend is only possible due to the monumental engineering
 effort of the Jupyter Project.
 
+## Development
+
+To run Jute locally, you must supply it with access to a Jupyter kernel.
+Currently, this is done by supplying environment variables to tell Jute where to
+find a Python executable and a
+[Jupyter data directory](https://docs.jupyter.org/en/stable/use/jupyter-directories.html#data-files).
```

Comment:
BTW this is crazy but I'm considering… bundling `uv` in Jute? After mulling about it for a while it seems like detecting local Python environments is kind of a lost cause and maybe it's possible to just create the Python environments within the Jute appdata folder? 🤔

Reply:
definitely a great idea, I was going to recommend something similar. full dependency isolation is also an interesting idea here - if you don't have to deal with conflicting python packages, python versions, etc per notebook and can instead control your execution environment, that'd be really nice. fits in nicely too with the design goal of "double click a notebook and expect it to work"

there was this project I came across a while ago that looked interesting: https://github.com/manzt/juv

but regardless, the industry is definitely moving towards uv as the standard so this feels like a good bet to make